trade-finance-manager-ui/server/controllers/utilisation-reports/remove-fees-from-payment/index.test.ts

```diff
@@ -23,20 +24,49 @@ describe('controllers/utilisation-reports/remove-fees-from-payment', () => {
       jest.resetAllMocks();
     });
 
-    it('removes the selected fee records from the payment', async () => {
+    it('sets the request session editPaymentFormValues field', async () => {
```

Comment:
Minor comment but I think this should be included in the redirect test e.g. `redirects to '/utilisation-reports/:reportId/edit-payment/:paymentId' with the edit payment form values when removing fee records is successful`